common/src/api/internal/shared.rs

```diff
@@ -154,10 +154,29 @@ pub enum SourceNatConfigError {
 
 // We alias [`RackNetworkConfig`] to the current version of the protocol, so
 // that we can convert between versions as necessary.
-pub type RackNetworkConfig = RackNetworkConfigV1;
+pub type RackNetworkConfig = RackNetworkConfigV2;
```

Comment:
Can we have another type alias for `PortConfig = PortConfigV2` here too?

Reply:
Done.